main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -29,6 +30,16 @@ object FlixPackageManager {
 
   // TODO: Move functionality from "Packager" in here.
 
+  //TODO: download the correct version, not just the latest
+  //TODO: report errors
+  //TODO: tests
+  def installAll(manifest: Manifest, path: Path): Result[Unit, PackageError] = {
+    val flixDeps = findFlixDependencies(manifest)
+    println(flixDeps)
+    flixDeps.foreach(dep => install(s"${dep.username}/${dep.projectName}", path, null))
```

Comment:
This should do one a time and fail on the failure.

And also print some information to a print stream argument.